From d4bbfdbb79dadc8c7cfc89687dfb8870f1f35bca Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 5 Dec 2023 15:32:15 +0100
Subject: [PATCH 0712/1141] arm64: dts: st: fix dsi node in stm32mp257f-ev1

These lines have to be remove to avoid warnings.

Change-Id: I3844c14fe40e0aa59287837102e036072eab10b7
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/344050
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 183357878078..7051cdd884d3 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -194,8 +194,6 @@ dcmipp_0: endpoint {
 };
 
 &dsi {
-	#address-cells = <1>;
-	#size-cells = <0>;
 	status = "disabled";
 
 	ports {
-- 
2.39.2

